test/correctness/mask_to_bool_scalar.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Halide.h"
+
+using namespace Halide;
+
+int main() {
```

Comment:
Per the Travis failure, this test needs to check if Hexagon is enabled in the target and if not, print a warning and exit with a success (0) indication.